webapp/client/src/components/FormNavButtons.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Row = styled.div`
+  margin-top: var(--spacing-09);
+`;
+
+const Button = styled.button`
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem;
+
+  font-size: var(--text-base);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--inverse-text-color);
+  background: var(--link-color);
+  display: inline-block;
+  position: relative;
+  border: none;
+  border-radius: 0;
+  border-bottom: 4px solid var(--link-color);
+
+  &:not(:disabled):not(.disabled):active {
+    background: var(--link-color) !important;
+    border: none !important;
+    border-bottom: 4px solid var(--link-color) !important;
+  }
+
+  &:hover {
+    background: var(--link-hover-color);
+    border: none;
+    border-bottom: 4px solid var(--link-hover-color);
+  }
+
+  &:focus {
+    color: var(--focus-text-color);
+
+    background: var(--focus-color);
+
+    outline: none;
+    box-shadow: none;
+    border: 0;
+    border-bottom: 4px solid var(--focus-border-color);
+  }
+`;
+
+const OutlineButton = styled.button`
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+
+  font-weight: var(--font-bold);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--link-color);
+
+  background: white;
+  background-clip: padding-box;
+
+  border: 0;
+  border-radius: 0;
+  border: 1px solid var(--border-color);
+
+  :not(:disabled):not(.disabled):active {
+    color: var(--link-active-color);
+    background-color: inherit;
+    border: 1px solid var(--link-active-color);
+  }
+
+  :hover {
+    color: var(--link-hover-color);
+    background: white;
+    border: 1px solid var(--link-hover-color);
+  }
+
+  :focus {
+    color: var(--focus-color);
+
+    outline: none;
+    box-shadow: none;
+    border: 1px solid var(--focus-border-color);
+  }
+`;
+
+const ExplanatoryText = styled.small`
+  margin-bottom: 0;
+  margin-left: var(--spacing-02);
+
+  & a {
+    color: var(--text-color);
+    font-weight: var(--font-bold);
+  }
+`;
+
+export default function FormNavButtons({
+  explanatoryButtonText,
+  overviewButton,
+  nextButtonLabel,
+}) {
+  // TODO intl
+  const backToOverviewText = "form.back_to_overview";
+  // TODO intl
```

Comment:
You probably can ignore this. I was just wondering what you mean with `intl`?

Reply:
"internationalize" - I had to look it up in Urban Dictionary, too. :D
(aka, use Babel for it^^)

Reply:
I was wondering if it was something in that direction. Thank you.

Reply:
Sorry, yes. This is a reminder for me that I've left some literal strings here that are just the translation keys from the Python app, which I'll have to stick into some kind of i18n tool. Probably [react-intl](https://formatjs.io/docs/react-intl/) or [i18next](https://react.i18next.com/).